acceptance/bundle/templates/helpers-error/script

```diff
@@ -0,0 +1,2 @@
+errcode trace $CLI bundle init .
+# test runner will complain if helpers.txt is found - it should not be there
```

Comment:
What does this comment refer to? I don't follow.

Reply:
Good catch, that was a copy paste error. Cleaned up.